src/Controls/src/Core/Menu/MenuFlyoutItem.cs

```diff
@@ -1,12 +1,22 @@
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.ComponentModel;
 using System.Windows.Input;
 using Microsoft.Maui.Controls.StyleSheets;
 
 namespace Microsoft.Maui.Controls
 {
-	public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
-	{
-	}
+    public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
+    {
+		public MenuFlyoutItem()
+		{
+			KeyboardAccelerators = new ObservableCollection<KeyboardAccelerator>();
```

Comment:
I see no way for adding an item to this list to trigger a platform change, so I wonder if this is sufficient:

```cs
this. KeyboardAccelerators.CollectionChanged += (sender, e) =>
    this.OnPropertyChanged(this, new(nameof(KeyboardAccelerators)));
```

MenuItem is a BO so a property change event will trigger the mappers... Will need a test.